po/en.po

```diff
@@ -256,10 +262,13 @@ msgid ""
 "때문에 특정 사용자가 리소스를 장기간 점유하는 문제를 해결하기에 어려움이 있습"
 "니다. 따라서 사용자 여러분들의 배려가 필요합니다."
 msgstr ""
+"Since the A100 GPU resources are limited, please only use it when necessary, such as when having a large deep learning model size or urgent training needs."
+"Unlike external clouds or Seoul National University's College of Engineering's 'GPU First', all services are provided for free. Therefore, it is challenging to address the issue of specific users occupying resources for extended periods."
```

Comment:
```suggestion
"Unlike external clouds or SNU College of Engineering's 'GPU First', all services are provided for free. Therefore, it is challenging to prevent some bad users from occupying resources for a long time."
```